src/battle_ai_main.c

```diff
@@ -1739,6 +1739,15 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
             break;
         case EFFECT_HIT_ESCAPE:
             break;
+        case EFFECT_CHILLY_RECEPTION:
+            if (CountUsablePartyMons(battlerAtk) == 0)
+                ADJUST_SCORE(-10);
+            else if (weather & (B_WEATHER_SNOW | B_WEATHER_PRIMAL_ANY)
+             || IsMoveEffectWeather(aiData->partnerMove))
```

Comment:
```suggestion
            else if (weather & (B_WEATHER_SNOW | B_WEATHER_PRIMAL_ANY) || IsMoveEffectWeather(aiData->partnerMove))
```

Reply:
For my own knowledge for future reviewing, is there a threshold beyond which we prefer multiline conditions vs single line conditionals?

Reply:
2 conditions can for sure go on one line, after that it is up to reviewer.

Reply:
How I did it is consistent with Snowscape.

Reply:
You can change that as well if you want but how Snowscape didn't shouldn't generally matter here. It is really just a nitpick :sweat_smile:

Reply:
Ok, done